doc/040_backup.rst

```diff
@@ -45,6 +45,17 @@ size of the files and directories in ``~/work`` on the local file system. It
 also tells us that only 1.200 GiB was added to the repository. This means that
 some of the data was duplicate and restic was able to efficiently reduce it.
 
+We just attached the absolute path ``~/work`` to the backup, so the path
```

Comment:
What about a plain `We backed up the absolute path ..., so the snapshot contains the full path:` followed by the output of `ls`?

Reply:
I personally think documentation should contain a path like` /home/user/work` as example for an absolute path.